<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTerminalDlg</class>
 <widget class="QDialog" name="AddTerminalDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加终端</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>监测点名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leMonitorName"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>监测点地址</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="leMonitorAddr"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>上一个监测点</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbPreMonitor"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>A</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>基础信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="3" column="4">
           <widget class="QLineEdit" name="leRoute61"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_59">
            <property name="text">
             <string>终端地址</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_60">
            <property name="text">
             <string>路由6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leRoute11"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leAddr1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_61">
            <property name="text">
             <string>路由1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leRoute31"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_62">
            <property name="text">
             <string>路由2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_63">
            <property name="text">
             <string>中继次数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_64">
            <property name="text">
             <string>路由5</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="leRoute21"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_65">
            <property name="text">
             <string>路由4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_66">
            <property name="text">
             <string>路由3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="leRoute41"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leRoute51"/>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="leTimes1"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string> 变比</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="leHighPressureValue1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leHighPressureOffset1">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>变比值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>偏移量</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>B</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>基础信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="3" column="4">
           <widget class="QLineEdit" name="leRoute62"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_51">
            <property name="text">
             <string>终端地址</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_52">
            <property name="text">
             <string>路由6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leRoute12"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leAddr2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_53">
            <property name="text">
             <string>路由1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leRoute32"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_54">
            <property name="text">
             <string>路由2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_55">
            <property name="text">
             <string>中继次数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_56">
            <property name="text">
             <string>路由5</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="leRoute22"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_57">
            <property name="text">
             <string>路由4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_58">
            <property name="text">
             <string>路由3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="leRoute42"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leRoute52"/>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="leTimes2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string> 变比</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLineEdit" name="leHighPressureValue2">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>变比值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leHighPressureOffset2">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>偏移量</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>C</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>基础信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="4">
           <widget class="QLineEdit" name="leRoute63"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>终端地址</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_50">
            <property name="text">
             <string>路由6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leRoute13"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leAddr3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>路由1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leRoute33"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>路由2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>中继次数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>路由5</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="leRoute23"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>路由4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>路由3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="leRoute43"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leRoute53"/>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="leTimes3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string> 变比</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>变比值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leHighPressureValue3">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leHighPressureOffset3">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>偏移量</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
